_includes/schema_org.html

```diff
@@ -0,0 +1,39 @@
+    <!-- Schema.org annotations in JSON-LD for a CreativeWork type.
+         Variables should be declared in the .md file 
+         For full list of available attributes see https://schema.org/CreativeWork -->
+    <script type="application/ld+json">
+    {
+       "@context": "http://schema.org/",
+       "@type": "creativeWork",
+       "genre": "trainingMaterial",
+       "name": "{{site.data.lesson.title}}",
+       "description": "{{site.data.lesson.description}}",
+       "keywords": "{{site.data.lesson.keywords}}",
+       "audience": [
+          {% for audience in site.data.lesson.audience %}{
+           "@type": "Audience",
+           "name": "{{audience}}"
+          }{% if forloop.last %}{%else%},{%endif%}
+          {% endfor %}
+       ],
+       "license": "{{site.data.lesson.licence}}",
+       "author": [
+         {% for author in site.data.lesson.author %}{
+           "@type": "Person",
+           "name": "{{author}}"
+          }{% if forloop.last %}{%else%},{%endif%}
+          {% endfor %}
+      ],
+       "contributor": [
+         {% for contributor in site.data.lesson.contributor %}{
+           "@type": "Person",
+           "name": "{{contributor}}"
+          }{% if forloop.last %}{%else%},{%endif%}
+          {% endfor %}
```

Comment:
Author and contributor are vague terms as lessons are being developed by the community. There is an initial author and then there is an army of contributors. Giving proper credits is important, but it should probably happen on a separate page

Reply:
The distinction between author and contributor:
Author: Preparation, creation and/or presentation of the published work, specifically writing the initial draft;
Contributor: a critical review, commentary or revision;
We need this information in metadata. We may reuse it to display on the separate page.